networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/Eth2P2PNetworkBuilder.java

```diff
@@ -139,6 +144,18 @@ public Eth2P2PNetwork build() {
     // Setup eth2 handlers
     final SubnetSubscriptionService attestationSubnetService = new SubnetSubscriptionService();
     final SubnetSubscriptionService syncCommitteeSubnetService = new SubnetSubscriptionService();
+
+    // TODO a bit hacky solution, subject to be refactored
```

Comment:
also would prefer to not have 'hacky solution' in a comment...

Reply:
It's draft yet, it was PoC, I wanted to check if it works first, because it's not very straightforward